src/components/Board/index.spec.js

```diff
@@ -1492,6 +1493,387 @@ describe('<Board />', () => {
+
+            it('calls the "onCardNew" callback passing the updated board, the updated column and the new card', () => {
+              expect(onCardNew).toHaveBeenCalledTimes(1)
+              expect(onCardNew).toHaveBeenCalledWith(
+                {
+                  columns: [
+                    {
+                      id: 1,
+                      title: 'Column Backlog',
+                      cards: [
+                        {
+                          id: 1,
+                          title: 'Card title 1',
+                          description: 'Card content'
+                        },
+                        {
+                          id: 2,
+                          title: 'Card title 2',
+                          description: 'Card content'
+                        },
+                        { id: 999, title: 'Card title', description: 'Card description' }
+                      ]
+                    },
+                    {
+                      id: 2,
+                      title: 'Column Doing',
+                      cards: [
+                        {
+                          id: 3,
+                          title: 'Card title 3',
+                          description: 'Card content'
+                        }
+                      ]
+                    }
+                  ]
+                },
+                {
+                  id: 1,
+                  title: 'Column Backlog',
+                  cards: [
+                    {
+                      id: 1,
+                      title: 'Card title 1',
+                      description: 'Card content'
+                    },
+                    {
+                      id: 2,
+                      title: 'Card title 2',
+                      description: 'Card content'
+                    },
+                    { id: 999, title: 'Card title', description: 'Card description' }
+                  ]
+                },
+                expect.objectContaining({ id: 999 })
+              )
+            })
+          })
+
+          describe('about the card position when the it is added', () => {
+            describe('when the position is not specified', () => {
+              beforeEach(async () => {
+                mount({ allowAddCard: true, onNewCardConfirm, onCardNew })
+                fireEvent.click(subject.queryAllByText('+')[0])
+
+                fireEvent.change(subject.container.querySelector('input[name="title"]'), {
+                  target: { value: 'Card title' }
+                })
+                fireEvent.change(subject.container.querySelector('input[name="description"]'), {
+                  target: { value: 'Card description' }
+                })
+                fireEvent.click(subject.queryByText('Add'))
+                await waitForElement(() => subject.container.querySelector('[data-testid="card"]:nth-child(3)'))
+              })
+
+              it('calls the "onNewCardConfirm" passing the new card', () => {
```

Comment:
I don't know whether a miss something above, but I think this context is just about the position, not about the adding. What do you think to keep this clean talking just about the positions?